resources/assets/lib/stores/commentable-meta-store.ts

```diff
@@ -16,7 +16,7 @@ export default class CommentableMetaStore {
     this.meta.clear();
   }
 
-  get(type: string, id: number) {
+  get(type: string, id: number): CommentableMetaJson | undefined {
```

Comment:
the implicit return type is the intersection of the `CommentableMetaJson` union, `{ title: string; }`. I don't really understand why...